README.md

```diff
@@ -202,6 +202,14 @@ To upgrade to a specific version, run the following command:
 luarocks install --server=http://luarocks.org/manifests/moesif kong-plugin-moesif 2.0.2
 ```
 
+### Reload Kong
+
+After upgrading the plugin, you must reload Kong to load the new version of the plugin.
+
+```bash
+kong reload
```

Comment:
@bakennedy should we refer our kong restart here.
- this suggestion is everywhere in readme to create confusion and doc long.
- any change in future will happen at one place but reflected everywhere